### PR TITLE
Support zipkin collector endpoint configuration 

### PR DESCRIPTION
**Pull request by Kuromesi**
**Please provide a description of this PR:**

This PR aims to support customized endpoint of this feature request https://github.com/istio/istio/issues/53086.

### CODE REVIEW

**Review by howardjohn — APPROVED** [review]

**Comment by zirain** [comment]
need a release notes

**Comment by zirain** [comment]
wait for https://github.com/istio/istio/pull/53115?

**Comment by Kuromesi** [comment]
> wait for #53115?

I think so

File: pilot/pkg/networking/core/tracing_test.go [hunk 1]
```diff
@@ -46,6 +46,7 @@ func TestConfigureTracingExhaustiveness(t *testing.T) {
 func TestConfigureTracing(t *testing.T) {
 	clusterName := "testcluster"
 	authority := "testhost"
+	defaultZipkinEndpoint := "/api/v2/spans"
```
Comment (zirain) [inline pilot/pkg/networking/core/tracing_test.go:49]: can you move this to a const, so that it could be shared in two changed files.

Comment (Kuromesi) [reply pilot/pkg/networking/core/tracing_test.go]: Done

**Review by zirain — APPROVED** [review]

**Pull request merged**